tests/unittests/test_boolean_conversion.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+from tap_google_sheets.singer_transform import transform
+
+class TestBooleanDataType(unittest.TestCase):
+    def test_string_returned_for_non_boolean_columns(self):
+        record = {
+            "__sdc_row": 1,
+            "Boolean": "string",
+        }
+        schema = {
+            "properties": {
+                "__sdc_row": {
+                    "type": [
+                        "integer",
+                        "null"
+                    ]
+                },
+                "Boolean": {
+                    "type": [
+                        "boolean",
+                        "string",
+                        "null"
+                    ]
+                }
+            },
+            "type": "object",
+            "additionalProperties": False
+        }
+        metadata = {(): {
```

Comment:
Move these `schema` and `metadata` out of class(global as those are used by all unit tests.

Reply:
declared it as global variables